linux_os/guide/services/ssh/ssh_server/sshd_use_approved_kex_ordered_stig/ansible/shared.yml

```diff
@@ -1,4 +1,4 @@
-# platform = Red Hat Enterprise Linux 7,Oracle Linux 7
+# platform = Red Hat Enterprise Linux 7,Oracle Linux 7,multi_platform_sle
```

Comment:
suggestion to change :
```
{{% set kex_algos_regex=prefix_conf ~ "(?=[\w-])" "(\\b" ~ kex_algos|join("\\b,?)?(\\b") ~ ")?" ~
sufix_conf ~ "[\s]*(?:#.*)?$" %}}
```
to
```
{{% set kex_algos_regex=prefix_conf %}}

```

Reply:
Thanks @teacup-on-rockingchair for the suggestion, I've made the requested changes!